examples/meta/CMakeLists.txt

```diff
@@ -25,6 +25,10 @@ LIST(APPEND GENERATOR_DEPENDENCIES ${TARGET_LANGUAGES})
 
 # parser output
 file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/parser_files)
+execute_process(COMMAND ${PYTHON_EXECUTABLE}
```

Comment:
note from cmake manual: "(execute_process) run while CMake is processing the project prior to build system generation."
i'm not so sure we wanna have this.... because you wanna run this when you build the target, i.e. the meta examples :)
for that use `add_custom_target` or `add_custom_command` and make sure that whoever depends on the output of `parse.py` depends on this new target/command. dont worry cmake will make sure that this target will be only called once and not as many times somebody depends on it.

Reply:
there are couple of examples within shogun cmake files how to use add_custom_target or add_custom_command, but if you are in doubt ask :)